test/grpc/sessions.test.js

```diff
@@ -98,3 +98,50 @@ test.cb('findAll should check for valid account_id', (t) => {
     t.end()
   })
 })
+
+test.serial('findAll should limit the rows return', async (t) => {
+  const Sessions = promisifyAll(t.context.clients.Sessions)
+  const payload = [
+    { name: 'app_open', timestamp: dayjs().unix() * 1000, ...app_open },
```

Comment:
1000 is used to convert dayjs().unix to unix timestamp. Instead of writing * 1001 please add 1 minute for each line. Example: dayjs().add(1, "day").unix() * 1000